WordPress/WordPressTest/Mention/SuggestionsListViewModelTests.swift

```diff
@@ -115,11 +115,39 @@ class SuggestionsListViewModelTests: CoreDataTestCase {
         XCTAssertTrue(result)
     }
 
+    /// Tests that the seach result has one match with case-insensitive and diacritic-insensitive
```

Comment:
Just noticed this small typo (and you got the same in other test cases below)
```suggestion
    /// Tests that the search result has one match with case-insensitive and diacritic-insensitive
```

Reply:
Fixed. Thank you!